### PR TITLE
Initial documentation for processing rules

### PR DESCRIPTION
**Pull request by Dalje-et**
### Merge instructions

- [X] Please merge after reviewing

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Hi! @Dalje-et, I've opened [a docs card](https://datadoghq.atlassian.net/browse/DOCS-9054) for this one. Someone on the docs team will give you a review and help wire this up to the nav etc.

File: content/en/service_management/on-call/processing_rules.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  text: 'Datadog On-Call'
+---
+
+Processing Rules allow teams to customize their response strategies for distinct types of incoming events, enabling them to orchestrate events to escalation policies and urgency levels based on the event's metadata. Low urgency pages do not trigger escalation processes.
+
+Datadog creates a default processing rule when you [onboard a Team to On-Call][1].
+
+### Viewing processing rules of your team
+
+To see the processing rule of your On-Call Team, simply click on its name in the [list of Teams][2].
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:15]:
```suggestion
To see the processing rule of your On-Call Team, click on the Team name in the [list of Teams][2].
```

File: content/en/service_management/on-call/processing_rules.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Processing Rules
+further_reading:
+- link: '/service_management/on-call/'
+  tag: 'Documentation'
+  text: 'Datadog On-Call'
+---
+
+Processing Rules allow teams to customize their response strategies for distinct types of incoming events, enabling them to orchestrate events to escalation policies and urgency levels based on the event's metadata. Low urgency pages do not trigger escalation processes.
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:9]:
```suggestion
## Overview

On-call processing rules allow teams to customize their response strategies for distinct types of incoming events. This enables teams to add events and urgency levels to to escalation policies based on the event's metadata. Low urgency pages do not trigger escalation processes.
```

File: content/en/service_management/on-call/processing_rules.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Datadog creates a default processing rule when you [onboard a Team to On-Call][1].
+
+### Viewing processing rules of your team
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:13]:
```suggestion
## View your team's processing rules
```

File: content/en/service_management/on-call/processing_rules.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+To see the processing rule of your On-Call Team, simply click on its name in the [list of Teams][2].
+
+### Query Syntax
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:17]:
```suggestion
## Query syntax
```

File: content/en/service_management/on-call/processing_rules.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Processing rule follow the common query syntax of Datadog. Supported attributes include:
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:19]:
```suggestion
Processing rules follow the common query syntax of Datadog. Supported attributes include:
```

File: content/en/service_management/on-call/processing_rules.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+* `tags`: the tags set on the incoming alert. For example, `tags.env:prod`.
+* `groups`: checks if the incoming alert relates to a specific Monitor group. For example, `groups:"service:checkout-service"`.
+* `priority`: value of the priority field of the Monitor. Possible values include 1, 2, 3, 4, or 5. For instance, `priority:(1 OR 2)`.
+* `alert_status`: value of the Monitor's status. Possible values include `error`, `warn`, `success`. Usage example: `alert_status:(error OR warn)`.
+
+If no specific filter should be applied, use `*`.
+
+### Ordering
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:28]:
```suggestion
## Ordering
```

File: content/en/service_management/on-call/processing_rules.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+Ordering of Processing Rules matters. The system goes from top to bottom and will stop matching rules if one matches. If no query or time filter matches the incoming alert, the default processing rule will be used.
```
Comment (estherk15) [inline content/en/service_management/on-call/processing_rules.md:30]:
```suggestion
The ordering of processing rules matters. The system goes from top to bottom and stops at the first matching rule. If no query or time filter matches the incoming alert, the default processing rule is used.
```

File: content/en/service_management/on-call/processing_rules.md (hunk 2 above)
Comment (estherk15) [reply content/en/service_management/on-call/processing_rules.md:9]: I'm not sure what this means:
> enabling them to orchestrate events to escalation policies and urgency levels based on the event's metadata

**Review by estherk15 — APPROVED** [review]

**Pull request merged**